### PR TITLE
[clojure-macros/pt_br] Fixing references links positioning

### PR DESCRIPTION
**Pull request by davelima**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

Links were positioned inside a code block, preventing markdown
to parse links to HTML version

### CODE REVIEW

**Comment by davelima** [comment]
Hey @vendethiel any comment about this?

File: pt-br/clojure-macros-pt.html.markdown [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: clojure
 filename: learnclojure-pt.clj
 contributors:
     - ["Adam Bard", "http://adambard.com/"]
+    - ["David Lima", "https://github.com/davelima"]
```
Comment (vendethiel) [inline pt-br/clojure-macros-pt.html.markdown]: I'm sorry, but we reserve the contributor tag to significant contributors

**Comment by davelima** [comment]
Oh, sure!

Updated :smile:

**Comment by vendethiel** [comment]
Thanks & sorry for the delay!

**Pull request merged**